README.md

```diff
@@ -63,18 +63,24 @@ annotate
 ```
 Check out their Github page for more running options.
 
+### Architectural Constraints
+Cost is a limitation of this app. We are hosting on a free Heroku instance and need to keep it this way. Do not add additional components that could potentially add costs to hosting (e.g. databases, frameworks, etc). Please talk to us if you have questions.
```

Comment:
What do you think of a little softening of the wording?

> In submitting features or bug fixes, please do not add new infrastructure components—e.g. databases, message queues, external caches—that might increase operational hosting costs. Come talk to us if you have questions!

Might add information on where to talk to you (link to Slack? open an issue here?).

Reply:
I like it - will update. Thanks for the feedback @robbkidd